codegen/src/main/java/software/amazon/awssdk/codegen/poet/model/MemberCopierSpec.java

```diff
@@ -482,4 +350,17 @@ private String memberParamName() {
         }
         return Utils.unCapitalize(memberModel.getC2jShape()) + "Param";
     }
+
+    private static final class VariableSource {
```

Comment:
This class name is too generic for me, I'd prefer if it was renamed to be more specific to its task. 
IDK if I love disambiguator but it's the only think I can think of now :-)

Reply:
I can't think of a better name. It provides unique variable names... UniqueVariableSource? VariableUniquifier?

Reply:
UniqueVariableSource is better than VariableSource